ozi.cc

```diff
@@ -172,6 +172,9 @@ ozi_open_io(const QString& fname, QIODevice::OpenModeFlag mode)
 static void
 ozi_close_io()
 {
+  if (!stream) {
```

Comment:
This makes the check around the call to ozi_close_io in ozi_openfile redundant (line 263).